src/sections/target/Safety/SafetyTables.js

```diff
@@ -250,29 +236,17 @@ const etoxColumns = [
 ];
 
 const ReferencesCell = ({ children }) => {
-  const classes = useStyles();
-
   return children
     .filter(reference => reference.pubmedId || reference.refLink)
     .map(({ pubUrl, refLabel, pubmedId }) => {
       const isHecatos = pubUrl.indexOf('hecatos') !== -1;
+      const heCaTosUrl = 'https://cordis.europa.eu/project/id/602156/reporting';
       return (
         <React.Fragment key={`${refLabel}:${pubUrl}`}>
-          <Link external to={pubUrl}>
-            <Tooltip
-              classes={{ tooltip: classes.tooltip }}
-              title={
-                isHecatos
-                  ? "HeCaToS Deliverable D01.5 (2015) funded by 'EU 7th Framework Programme (HEALTH-F4-2013-602156)'."
-                  : ''
-              }
-              placement="top"
-            >
-              <span>
-                {refLabel ||
-                  (pubmedId ? `Pubmed ID: ${pubmedId}` : '(no name)')}
-              </span>
-            </Tooltip>
+          <Link external to={isHecatos ? heCaTosUrl : pubUrl}>
+            <span>
```

Comment:
Could we remove this span? I believe it was needed because [Tooltip](https://material-ui.com/api/tooltip/#props) requires its children to be a node, but now that Tooltip is removed I think we can remove this span as well.

Reply:
Thanks @mirandaio - that span element was redundant and only used previously when there was the tooltip.